contracts/src/Assets.sol

```diff
@@ -107,9 +106,6 @@ library Assets {
                 revert Unsupported();
             }
         } else {
-            if (destinationChainFee == 0) {
-                revert InvalidDestinationFee();
-            }
```

Comment:
Just curios why remove this check?

Reply:
Oops, that was made in error somehow. I will revert